examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
why only first 5 epochs? and you allow picks which is more than the number of epochs? How does that work?